tensor_comprehensions/pybinds/tc.cc

```diff
@@ -287,24 +287,16 @@ PYBIND11_MODULE(tc, m) {
           " 5. group_conv\n"
           " 6. single_thread")
       .def(
-          "maxSharedMemory",
-          &tc::CudaMappingOptions::maxSharedMemory,
-          "The amount of shared memory to use, in bytes. If not provided, "
-          "TC will query the active GPU and use all available shared memory.")
-      .def(
-          "useSharedMemory",
-          &tc::CudaMappingOptions::useSharedMemory,
-          "Create block-local copies of data in shared memory when this can "
-          "leverage data reuse or global memory access coalescing")
-      .def(
-          "unrollCopyShared",
-          &tc::CudaMappingOptions::unrollCopyShared,
-          "Also unroll the copies to and from shared memory. If an unroll "
-          "value is not provided, has no effect")
-      .def(
-          "useReadOnlyCache",
-          &tc::CudaMappingOptions::useReadOnlyCache,
-          "Use the readonly cache (i.e. emit __ldg loads)")
+          "toString",
```

Comment:
python has magic methods `__str__` and `__repr__` for this

Reply:
They just print things like:
```
<method-wrapper '__str__' of tensor_comprehensions.tc.CudaMappingOptions object at 0x7f715d879420>
```

We want a human readable form of the options instead which is what `toString` does (and it was already present before btw)

Reply:
that's why you want to _define_ `__str__` rather than some `toString`, it will print what you return (and so will `str(yourobject)`)

Reply:
ah yes of course, I am rusty with python, thanks!